routes/api/report.js

```diff
@@ -0,0 +1,88 @@
+var express = require('express');
+var router = express.Router();
+var utils = require('../../utils.js');
+var logger = require('log4js').getLogger();
+logger.level = 'debug';
+
+module.exports = function(connection) {
+
+    router.get('/report/module', function (req, res, next) {
+
+        connection.query('select data from atlas', function (error, rows, field) {
+
+            if(error) {
+                logger.error(error);
+            } else {
+                var resp = [];
+                var idxs = {};
+
+                rows.forEach(function(row) {
+                    var data = row.data;
+                    if(data && utils.isJson(data)) {
+                        data = JSON.parse(data);
+                        if(data.modules) {
+                            data.modules.forEach(function(module) {
+                                if(module.active !== "true") return;
+                                if(!idxs[module.name]) {
+                                    idxs[module.name] = resp.length;
+                                    resp.push({ id: module.name.toLowerCase(), name: module.name, versions: {} });
+                                }
+                                if(module.version) {
+                                    if(resp[idxs[module.name]].versions[module.version]) {
+                                        resp[idxs[module.name]].versions[module.version] += 1;
+                                    } else {
+                                        resp[idxs[module.name]].versions[module.version] = 1;
+                                    }
+                                }
+                            });
+                        }
+                    }
+                });
+
+                res.setHeader('Content-Type', 'application/json');
+                res.json(resp);
+            }
+        });
+    });
+    
+    router.get('/report/module/:module_name', function (req, res, next) {
```

Comment:
Use `module_id` instead of name. Module ids are lowercase and will not (should not) contain spaces.